_layouts/help.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: main
+---
+{% capture slug_permalink %}/{{ page.slug }}{% endcapture %}
+{% assign parent_url = page.url | replace: slug_permalink %}
+{% assign related_questions = site.help | where_exp: "item", "item.url contains parent_url" | sort: 'order' %}
+{% assign this_page_parent = '' %}
+
+{% include help/nav_page_top_small.html %}
+{% include help/nav_page_top.html %}
+
+<div class="bg-white">
+  <div class="container cntnr-wide px2 pt2 pb0">
+    <div class="clearfix">
+      {% include help/nav_sidenav.html %}
+      <div class="sm-col sm-col-8 pr5">
```

Comment:
I noticed on mobile the main content area still had a large amount of padding applied to the right side on these pages.  I think if we switch this to a `sm-pr5` then on mobile it should span the entire width of the screen.